paasta_tools/cli/cmds/remote_run.py

```diff
@@ -52,50 +77,36 @@ def add_remote_run_args(parser):
         default=DEFAULT_SOA_DIR,
     )
     parser.add_argument(
-        '--json-dict',
-        help='When running dry run, output the arguments as a json dict',
+        '-v', '--verbose',
+        help='Show Docker commands output',
         action='store_true',
-        dest='dry_run_json_dict',
-    )
-    parser.add_argument(
-        '-C', '--cmd',
-        help=('Run Docker container with particular command, '
-              'for example: "bash". By default will use the command or args specified by the '
-              'soa-configs or what was specified in the Dockerfile'),
         required=False,
-        default=None,
+        default=False,
     )
     parser.add_argument(
-        '-i', '--instance',
-        help=("Simulate a docker run for a particular instance of the service, like 'main' or 'canary'"),
+        '-R', '--run-id',
```

Comment:
Can we re-use instance for this? ("make" an instance on the fly?)

Because if people are saying "--instance main", what does that really mean in the context of local-run? On paasta logs, status, and sfx things will show up under "main", *not* under this run-id var.